docs/fedora-releases-guide/non-divergent-dist-git-branches.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Non-divergent dist-git branches
+sidebar_position: 7
+---
+
+# Non-divergent dist-git branches
+
+To prevent your dist-git branches from diverging, you can use the following `dist_git_branches` syntax:
+
+```yaml
+  dist_git_branches:
+    rawhide:
+      fast_forward_merge_into: [fedora-branched]
+    epel-9: {}    
+```
+
+In this example, Packit runs the downstream synchronization process for the `rawhide` and `epel-9` branches as usual. Additionally, Packit opens a new pull request reusing the commit from `rawhide` for every `fedora-branched` branch, allowing it to be fast-forwarded when merging.
+
+You can learn more about the `dist_git_branches` syntax for `propose_downstream` [here](docs/configuration/upstream/propose_downstream#optional-parameters) and for `pull_of_upstream` [here](docs/configuration/downstream/pull_from_upstream#optional-parameters).
+
+:::warning How to Reconcile Divergent Branches
+
+If you are already using Packit, your branches may have diverged. You need to reconcile them before using the new `dist_git_branches` syntax. For the configuration example above and the current state of the branched Fedora releases, you need to:
```

Comment:
I would maybe add a note that they should adjust the releases, if they reconcile later on. Might produce surprising results, if they use different releases :D